flink-libraries/flink-state-processing-api/src/main/java/org/apache/flink/state/api/Savepoint.java

```diff
@@ -52,7 +53,7 @@ private Savepoint() {}
 	 * @param stateBackend The state backend of the savepoint.
 	 */
 	public static ExistingSavepoint load(ExecutionEnvironment env, String path, StateBackend stateBackend) throws IOException {
-		org.apache.flink.runtime.checkpoint.savepoint.Savepoint savepoint = SavepointLoader.loadSavepoint(path);
+		CheckpointMetadata savepoint = SavepointLoader.loadSavepoint(path);
```

Comment:
Maybe we need to update the variable name `savepoint` here.
And do we need to update the function name of `SavepointLoader#loadSavepoint`.
After the refractory, seems a little strange that we pass a loaded savepoint to a `CheckpointMetadata`